tests/lib/k8s/test_builders.py

```diff
@@ -9,11 +9,38 @@ class TestK8sBuilders(TestCase):
     def compiled_templates_mock(self):
         """To mock compiled templates (with handlebars)"""
 
-        def compiled_templates(parameters):
+        def check_anti_affinity_node_template(parameters):
             self.assertEqual(parameters, {"app": "app", "process": "web"})
-            return {"mock": "template"}
+            return {"mock: template_anti_affinity_node"}
```

Comment:
Do you want a set here ? 🙂  (same for all templates)
```suggestion
            return {"mock": template_anti_affinity_node}
```
```suggestion
            return "mock: template_anti_affinity_node"
```